src/StickyParallaxHeader.js

```diff
@@ -56,7 +38,7 @@ class StickyParallaxHeader extends Component {
   }
 
   spring = () => {
-    const scrollNode = this.scroll
+    const scrollNode = this.scroll && this.scroll.scrollTo ? this.scroll : this.scroll.getNode()
```

Comment:
extract it please to separate file e.g. utils/getSafelyScrollNode and sth like this if possible:

```js
const getSafelyScrollNode = (scrollNode) => return {
// before 0.62 
if (scrollNode && scrollNode.scrollTo) return scrollNode

// after 0.62
return scrollNode.getNode()
} 
```
it will be easier to support it in future

Reply:
and use this function here

Reply:
good idea ;)

Reply:
✅